nautobot_netbox_importer/diffsync/models/dcim.py

```diff
@@ -54,26 +54,15 @@ def setup(adapter: SourceAdapter) -> None:
             "role": fields.role(adapter, "dcim.rackrole"),
         },
     )
-    adapter.configure_model(
-        "dcim.cable",
-        fields={
-            "termination_a": "termination_a",
-            "termination_b": "termination_b",
-        },
-    )
+    adapter.configure_model("dcim.cable")
```

Comment:
why the fields are not needed anymore?

Reply:
Thanks to the refactoring, these fields are autodetected now:

https://github.com/nautobot/nautobot-app-netbox-importer/pull/123/files#diff-fe77f230ddce96ba9a9d7b349e5e7acfe3c00818cd1818c543d2c1f29504d39dR913